encrypted-mediatype.md

```diff
@@ -0,0 +1,98 @@
+# Encrypted media types
+
+To be able to protect the confidentiality of the data in layers, encryption of the layer data blobs can be done to prevent unauthorized access to layer data. Encryption is performed on the data blob of a by specifying a media type with the `+encrypted` suffix. For example, `application/vnd.oci.image.layer.v1.tar+encrypted` is an layer representation of an encrypted `application/vnd.oci.image.layer.v1.tar` layer. 
+
+## `+encrypted` media type and annotation definitions
```

Comment:
Defining things this way does not scale: https://github.com/opencontainers/image-spec/issues/791

Reply:
nod... 

@lumjjb  Perhaps you could add an explanation at the top something to the effect that **this is a work in progress (WIP) because the `+ compression/encoding` suffix model is not seen as a scaleable model and provide link(s) to the discussion underway.** For that matter we could place it in a WIP section.

Reply:
Agreed on this, I've added the note that the mediatype definition is subject to change.